qemu/tests/cfg/rss_iperf_test.cfg

```diff
@@ -1,36 +1,37 @@
 - rss_iperf_test:
-    only Windows
-    # TODO: Currently virtio-net driver on Linux does not support RSS.
-    # Development working on that, no ETA yet.
-    only virtio_net
     virt_test_type = qemu
-    type = iperf_test
-    required_qemu = [5.1.0-2,)
     image_snapshot = yes
     firewalld_service = disable
+    type = iperf_test
+    required_qemu = [9.0.0-1,)
     fw_stop_cmd = systemctl stop firewalld || service iptables stop
     linux_compile_cmd = tar zxf %s -C %s > /dev/null ; cd %s ; ./configure > /dev/null; make > /dev/null
     check_iperf_cmd = pgrep -f %s
-    guest_iperf_path = 'c:\'
-    fw_stop_cmd = netsh advfirewall set allprofiles state off
-    check_iperf_cmd = wmic process where name='%s' list
+    only virtio_net
     disable_pci_msi = no
     catch_data = '%s port 5201 connected to %s'
     iperf_version = iperf-3.1.3
     host_iperf_file = ${iperf_version}.tar.gz
-    del vcpu_maxcpus
-    smp = 4
-    guest_iperf_file = ${iperf_version}.exe
-    iperf_deplist = 'cyggcc_s-1.dll,cygwin1.dll'
-    queues = ${smp}
-    parallel_num = ${queues}
-    iperf_server_options = ' -s -1 -i 0'
-    iperf_client_options = ' -c %s -B %s -i 0 -f K -w 300K -P ${parallel_num} -t %s'
-    iperf_test_duration = 60
     vhost_nic1 = off
     nic_extra_params += ",rss=on,hash=on"
     rss_test = yes
-    rss_test_cmd = "WIN_UTILS:\netkvm-wmi.cmd rss"
+    check_iperf_cmd = pgrep -f %s
+    check_rss_state = ethtool -x %s
+    enable_rxhash_cmd = ethtool -K %s rxhash on
+    queues = 4
+    parallel_num = ${queues}
+    iperf_test_duration = 60
+    iperf_server_options = ' -s -1 -i 0'
+    iperf_client_options = ' -c %s -B %s -i 0 -f K -w 300K -P ${parallel_num} -t %s'
+    Windows:
+        required_qemu = [5.1.0-2,)
+        del vcpu_maxcpus
```

Comment:
You delete `vcpu_maxcpus` here, but no define `smp`. with the default cartesian configuration, seems it's a fixed value.

Reply:
Let me update it and set it equal to queues.

Reply:
Please help review it again, thanks a lot.